src/components/double-scene/double-scene-component.jsx

```diff
@@ -57,17 +51,25 @@ const DoubleSceneComponent = ({ sceneId, children, loaderOptions, sceneSettings,
             container: `scene-local-container-${sceneId}`,
             ...sceneSettings,
             viewingMode: 'local',
-            clippingArea: localExtent,
-            extent: localExtent,
+            spatialReference,
           });
           setViewLocal(_viewLocal);
         })
         .catch(err => {
           console.error(err);
         });
     }
-  },[map])
+  },[map]);
 
+  useEffect(() => {
+    if(viewLocal && spatialReference && countryExtent) {
+      const expandedCountryExtent = countryExtent.clone().expand(1.5); //add paddings around country borders
+      viewLocal.clippingArea = expandedCountryExtent;
+      viewLocal.extent = expandedCountryExtent;
+      viewLocal.goTo({ target: expandedCountryExtent, tilt: 40, }, { animate: false });
```

Comment:
The `tilt` is not being set (not sure if that's related to the `clippingArea` error (but I do not think so, since the first time we toggle the local scene the tilt is not updated). Maybe we can get into this on the next PR (regarding styles of the scene)

Reply:
I moved `viewLocal.goTo()` to the `viewLocal.when(() => {...})`, it should solve the problem for now